modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/SelectWithOption.tsx

```diff
@@ -82,7 +83,9 @@ export function SelectWithOption({
 					aria-label={ariaLabel}
 					disabled={disabled}
 					onClick={() => setDropdownActive((active) => !active)}
-					placeholder={Liferay.Language.get('choose-an-option')}
+					placeholder={
```

Comment:
@manuelecastro You can just pass the placeholder here like `placeholder={placeholder}` and define a default value for it in the line 52 doing `placeholder = Liferay.Language.get('choose-an-option')`.